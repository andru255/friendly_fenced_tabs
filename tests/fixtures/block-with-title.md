``` friendly_title='My custom title'
my example of block
```
