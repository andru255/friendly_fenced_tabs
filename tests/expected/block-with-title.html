<div class="friendly_fenced_tabs">
    <ul class="friendly_fenced_tabs__header">
        <li class="friendly_fenced_tabs__title">
            My custom title
        </li>
    </ul>
    <ul class="friendly_fenced_tabs__content">
        <li class="friendly_fenced_tabs__item">
            <pre><code>example of block</code></pre>
        </li>
    </ul>
</div>